tools/distpartitioning/parmetis_wrapper.py

```diff
@@ -50,10 +54,11 @@ def run_parmetis_wrapper(params):
     # Trigger pre-processing step to generate input files for ParMETIS.
     preproc_cmd = (
         f"mpirun -np {num_partitions} -hostfile {params.hostfile} "
-        f"python3 tools/distpartitioning/parmetis_preprocess.py "
+        f"python3 $DGL_HOME/tools/distpartitioning/parmetis_preprocess.py "
```

Comment:
Have you verified this works? do users need to add `//tools` or any other directories into `PYTHONPATH`?

Reply:
something like this ? https://github.com/dmlc/dgl/blob/c396942d068d8c8c181162b0c9af4391ca404962/tests/scripts/task_distributed_test.sh#L39